pkgs/applications/graphics/glimpse/default.nix

```diff
@@ -0,0 +1,197 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, substituteAll
+, pkgconfig
+, intltool
+, babl
+, gegl
+, gtk2
+, glib
+, gdk-pixbuf
+, isocodes
+, pango
+, cairo
+, freetype
+, fontconfig
+, lcms
+, libpng
+, libjpeg
+, poppler
+, poppler_data
+, libtiff
+, libmng
+, librsvg
+, libwmf
+, zlib
+, libzip
+, ghostscript
+, aalib
+, shared-mime-info
+, python2
+, libexif
+, gettext
+, xorg
+, glib-networking
+, libmypaint
+, gexiv2
+, harfbuzz
+, mypaint-brushes1
+, libwebp
+, libheif
+, libgudev
+, openexr
+, AppKit
+, Cocoa
+, gtk-mac-integration-gtk2
+, libxslt
+, automake
+, autoconf
+, libtool
+, makeWrapper
+, autoreconfHook
+, gtk-doc
+}:
+let
+  python = python2.withPackages (pp: [ pp.pygtk ]);
+in
+stdenv.mkDerivation rec {
+  pname = "glimpse";
+  version = "0.2.0";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    owner = "glimpse-editor";
+    repo = "Glimpse";
+    rev = "v${version}";
+    sha256 = "sha256-qbZQmAo7fuJWWbn0QTyxAwAenZOdsGueUq5/3IV8Njc=";
+  };
+
+  patches = [
+    # to remove compiler from the runtime closure, reference was retained via
+    # gimp --version --verbose output
+    (substituteAll {
+      src = ./remove-cc-reference.patch;
+      cc_version = stdenv.cc.cc.name;
+    })
+    ../gimp/hardcode-plugin-interpreters.patch
+  ];
+
+  postPatch = ''
+    ln -s ${gtk-doc}/share/gtk-doc/data/gtk-doc.make .
+  '';
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+    intltool
+    gettext
+    automake
+    autoconf
+    makeWrapper
+    gtk-doc
+  ];
+
+  buildInputs = [
+    babl
+    gegl
+    gtk2
+    glib
+    gdk-pixbuf
+    pango
+    cairo
+    gexiv2
+    harfbuzz
+    isocodes
+    freetype
+    fontconfig
+    lcms
+    libpng
+    libjpeg
+    poppler
+    poppler_data
+    libtiff
+    openexr
+    libmng
+    librsvg
+    libwmf
+    zlib
+    libzip
+    ghostscript
+    aalib
+    shared-mime-info
+    libwebp
+    libheif
+    python
+    libexif
+    xorg.libXpm
+    glib-networking
+    libmypaint
+    mypaint-brushes1
+    libxslt
+    libtool
```

Comment:
These should likely go to `nativeBuildInputs`.

Reply:
Done